HW_13_generator.py

```diff
@@ -0,0 +1,18 @@
+
+def my_gen(stop = 0):
```

Comment:
we do not need stop
the sequence should be infinite, in your case it is limited by stop

Reply:
we should accept sequence as a parameter